lib/plugins/inspec-init/templates/profiles/aws/controls/example.rb

```diff
@@ -3,24 +3,28 @@
 
 title 'Sample Section'
 
-aws_vpc_id = attribute('aws_vpc_id')
+aws_vpc_id = attribute('aws_vpc_id', default: '', description: 'Optional AWS VPC identifier.')
 
-# you add controls here
-control 'aws-vpc-check' do                                                  # A unique ID for this control.
+# You add controls here
+control 'aws-single-vpc-exists-check' do                                    # A unique ID for this control.
+  only_if { aws_vpc_id != ''}                                               # Only run this control if the `aws_vpc_id` atrtibute is provided.
   impact 1.0                                                                # The criticality, if this control fails.
-  title 'Check to see if custom VPC exists.'                                # A human-readable title
+  title 'Check to see if custom VPC exists.'                                # A human-readable title.
   describe aws_vpc(aws_vpc_id) do                                           # The test itself.
     it { should exist }
   end
 end
 
-# Plural resources can be inspected to check for specific resource details.
-control 'aws-vpcs-check' do
-  impact 1.0
-  title 'Check in all the VPCs for default sg not allowing 22 inwards'
-  aws_vpcs.vpc_ids.each do |vpc_id|
-    describe aws_security_group(vpc_id: vpc_id, group_name: 'default') do
```

Comment:
Aww, it saddens me that this test would be removed.  It's very practical, and shows the power of inspec as a security auditing tool. Is there a strong reason to remove this?

Reply:
Happy to replace this.